sourcecode/hub/resources/views/components/layout.blade.php

```diff
@@ -1,9 +1,152 @@
 <x-layout.base :nav="$nav ?? true">
-    <main class="container">
-        <header>
-            <h1 class="fs-2">{{ $title }}</h1>
-        </header>
+    <div class="container-fluid">
+        <div class="row">
+            <!-- Sidebar -->
+            <nav id="sidebar" class="col-md-3 col-lg-2 d-md-block bg-white sidebar">
+                <div class="position-sticky">
+                    <!-- Sidebar content -->
+                    <div class="accordion p-3 m-0 border-0 bd-example m-0 border-0" id="filterAccordion">
+                        <div class="accordion-item">
+                            <h2 class="accordion-header">
+                                <button id="language" class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                                    Language
+                                </button>
+                            </h2>
+                            <div id="collapseOne" class="accordion-collapse collapse show" data-bs-parent="#accordionExample">
+                                <div class="accordion-body p-0">
+                                    <ul class="list-group">
+                                        <li class="list-group-item border-0">
+                                            <input class="form-check-input me-1" type="checkbox" value="english" id="english">
+                                            <label class="form-check-label" for="firstCheckbox">English</label>
```

Comment:
The element that this is a description [for](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/for), `firstCheckbox`, does not exist. The `for="firstCheckbox` is set on several elements. Same for `secondCheckbox` and `thirdCheckbox`.
Prefer nesting the `input` element inside the `label` element over creating the association with `for` and `id` attributes.